components/AlphabetGlossary.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+    <div>
+      <div class="text-right mb-5" :style="{marginRight: '10%' }">
+        <button @click="expandAll" class="hover:text-gray-400 text-gray-800 font-bold py-2 px-1 underline text-xs">Expand All</button>
+        <button @click="collapseAll" class="hover:text-gray-400 text-gray-800 font-bold py-2 px-1 underline text-xs">Collapse All</button>
+      </div>
+      <div :style="{ marginLeft: '10%', marginRight: '10%' }">
```

Comment:
Another candidate for Tailwind instead of inline styles. I won't flag all of these so if you could check if there are any more that would be great thanks Ruby.